backend/README.md

```
@@ -6,7 +6,9 @@ project/
 ├─ api/
 │  └─ index.py          # FastAPI entrypoint
 ├─ .env                 # Supabase + Backend config (ignored by git)
-├─ requirements.txt     # Python dependencies
+├─ .env                 # Supabase + Backend config (ignored by git)
+├─ pyproject.toml       # Python dependencies and config
+├─ uv.lock              # Dependency lock file
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)